.travis.yml

```diff
@@ -1,17 +1,13 @@
-sudo: false
-dist: trusty
 language: rust
-
-before_install:
-  - cargo install mdbook --vers '0.0.25' --debug --force
-
+cache:
+- cargo
 script:
-  - mdbook build
-
+- mdbook build
+- mdbook test
```

Comment:
mdbook has a test functionality? Does it check for invalid configs or bad md?

Reply:
Apparently! The rustc-guide suggests as much.

Reply:
neat!